general/clients/skins.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: css-custom
+title: CSS Customization
+---
+
+# CSS Customization
```

Comment:
You'll need to add this page to the table of contents as well.

Reply:
done. added books as well.